fsm/src/main/java/org/egov/fsm/service/notification/NotificationService.java

```diff
@@ -0,0 +1,212 @@
+package org.egov.fsm.service.notification;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.fsm.config.FSMConfiguration;
+import org.egov.fsm.repository.ServiceRequestRepository;
+import org.egov.fsm.service.UserService;
+import org.egov.fsm.util.FSMConstants;
+import org.egov.fsm.util.NotificationUtil;
+import org.egov.fsm.web.model.FSM;
+import org.egov.fsm.web.model.FSMRequest;
+import org.egov.fsm.web.model.notification.ActionItem;
+import org.egov.fsm.web.model.notification.Event;
+import org.egov.fsm.web.model.notification.EventRequest;
+import org.egov.fsm.web.model.notification.Recepient;
+import org.egov.fsm.web.model.notification.SMSRequest;
+import org.egov.fsm.web.model.notification.Source;
+import org.egov.fsm.web.model.workflow.Action;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+
+import com.jayway.jsonpath.JsonPath;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Service
+public class NotificationService {
+
+	private FSMConfiguration config;
+
+	private ServiceRequestRepository serviceRequestRepository;
+
+	private NotificationUtil util;

```

Comment:
NotificationUtil is not autowired. Use only one type of autowiring

Reply:
Done